tools/nuget/generate_nuspec_for_native_nuget.py

```diff
@@ -286,7 +294,11 @@ def generate_metadata(line_list, args):
     generate_owners(metadata_list, "Microsoft")
     generate_description(metadata_list, args.package_name)
     generate_copyright(metadata_list, "\xc2\xa9 " + "Microsoft Corporation. All rights reserved.")
-    generate_tags(metadata_list, "ONNX ONNX Runtime Machine Learning")
+    generate_tags(
+        metadata_list, "ONNX ONNX Runtime Machine Learning"
+    ) if "Microsoft.ML.OnnxRuntime.Training." in args.package_name else generate_tags(
+        metadata_list, "ONNX ONNXRuntime-Training Learning-on-The-Edge On-Device-Training MachineLearning"
```

Comment:
nit: we should add 'native' to there as per nuget guidelines.

Also not sure that we need two 'ONNX' entries if the words aren't quoted.

Reply:
https://learn.microsoft.com/en-us/nuget/guides/native-packages

> Be sure to include native in the <tags> section of your .nuspec to help other developers find your package by searching on that tag.

Reply:
I added native, tags are space delimited: So I added ONNX and ONNXRuntime-Training